### ISSUE TITLE
KeyError: page

### ISSUE THREAD

**Issue by ggree1** [uriyyo/fastapi-pagination#895]
Firstly, thanks for the great library.
I have a problem after upgrading latest version of fastapi-pagination.

```python
class Params(BaseModel, AbstractParams):
    page: int = Query(1, ge=1, alias="pageNumber", description="Page number")
    size: int = Query(50, ge=1, le=100, alias="pageSize", description="Page size")

    def to_raw_params(self) -> RawParams:
        return RawParams(
            limit=self.size,
            offset=self.size * (self.page - 1),
        )
``` 

It causes KeyError: 'page' in 
../../../.venv/lib/python3.11/site-packages/fastapi_pagination/api.py:258: in _create_params_dependency
    sign_params[name] = sign_params[name].replace(default=field)
 
sign_params dict has aliases (pageNumber, pageSize), so it represents KeyError.
I have to remove alias or rename it to the same as variable name.

Is this a bug?
It occurrs after upgrading fastapi 1.x and pydantic v2, and fastapi-pagination recent version.

**Comment by uriyyo** [comment]
Hmmm, I guess that's because of how `pydantic v2` generates a signature for model. I will take a look.

**Comment by uriyyo** [comment]
@ggree1 This issue should be fixed in the new version `0.12.12`. Could you please verify it?

**Comment by ggree1** [comment]
> @ggree1 This issue should be fixed in the new version `0.12.12`. Could you please verify it?

Yes, it doesn't appear anymore with 0.12.12.
Thanks! :)
